sentences/nl/homeassistant_HassTurnOff.yaml

```diff
@@ -12,8 +12,9 @@ intents:
         response: "cover"
 
       - sentences:
-          - sluit <name> [in|op] <area>
-          - "[<doe>|<zou>] <name> (<dicht> [in|op] <area>|[in|op] <area> <dicht>) [willen|kunnen]\
-            \ [<doe>]"
-          - "<zou> <name> [in|op] <area> (willen|kunnen) [<dicht>]"
-        response: "cover_area"
+          - "sluit [de] garage[ ][deur]"
+          - "[<doe>|<zou>] [de] garage[ ][deur] dicht [willen|kunnen] [<doe>]"
+        response: "cover_device_class"
+        slots:
+          device_class: "garage"
+          domain: "cover"
```

Comment:
I'm confused, why is this cover sentence in `homeassistant` domain file ?

Reply:
Okay, with that in mind I will make some changes. But then there will be only one intent left :)